src/test/java/io/odh/test/e2e/continuous/DataScienceDashboardST.java

```diff
@@ -0,0 +1,115 @@
+@Tag(TestSuite.CONTINUOUS)
+//TODO: what is ST? system test?
+public class DataScienceDashboardST extends Abstract {
+    static final Logger LOGGER = LoggerFactory.getLogger(DataScienceDashboardST.class);
+
+    @Test
+    void checkDataScienceProjects() throws IOException, InterruptedException {
+        try (OpenShiftClient kubeClient = ResourceManager.getClient().getClient().adapt(OpenShiftClient.class)) {
+            Service service = kubeClient.services()
+                    .inNamespace("redhat-ods-applications")
+                    .withName("rhods-dashboard")
+                    .get();
+
+            ServicePort port1 = new ServicePortBuilder()
+                    .withName("8443-tcp")
+                    .withProtocol("TCP")
+                    .withPort(8443)
+                    .withTargetPort(new IntOrString(8443))
+                    .build();
+
+            ServicePort port2 = new ServicePortBuilder()
+                    .withName("8080-tcp")
+                    .withProtocol("TCP")
+                    .withPort(8080)
+                    .withTargetPort(new IntOrString(8080))
+                    .build();
+
+            Service updatedService = service.edit()
+                    .editSpec()
+                    .addToPorts(port1, port2)
+                    .endSpec()
+                    .build();
+
+            Route route = kubeClient.routes().load(new ByteArrayInputStream("""
+                    apiVersion: route.openshift.io/v1
+                    kind: Route
+                    metadata:
+                      name: rhods-dashboard-internal
+                      namespace: redhat-ods-applications
+                      labels:
+                        app: rhods-dashboard
+                        app.kubernetes.io/part-of: rhods-dashboard
+                    spec:
+                      port:
+                        targetPort: 8080-tcp
+                      to:
+                        kind: Service
+                        name: rhods-dashboard
+                      wildcardPolicy: None
+                    """.getBytes(StandardCharsets.UTF_8))).item();
+            kubeClient.routes().resource(route).create();
+
+            // do I need to wait here?
+
+            // fetch the route object from the server
+            Route r2 = kubeClient.routes().resource(route).get();
+            String url = "http://%s/api/status".formatted(r2.getSpec().getHost());
+
+            HttpRequest request = HttpRequest.newBuilder(URI.create(url))
+                    .GET()
+                    .header("x-forwarded-access-token", "asdfadsfasd")
+                    .build();
+            LOGGER.debug("Making request {}", request);
+            HttpClient httpClient = HttpClient.newHttpClient();
+            HttpResponse<Void> response = httpClient.send(request, HttpResponse.BodyHandlers.discarding());
+            LOGGER.debug("Got response {}", response);
```

Comment:
Regarding the test scenario itself. Maybe run this curl directly in the container the service is fronting? Or put my own container that contains `curl` there, and don't expose any routes...?